### PR TITLE
Fix up powershell and flaws

### LINKED ISSUE #1580: PowerShell command missing, only has CMD and does not tell you that its CMD only
While browsing on the [Express Tutorial Part 2 ](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/skeleton_website) I came across the part where we run a command to start the webpage.

Problem

The code here is correct for CMD but not for PowerShell. it should say 
windows 
     cmd : SET DEBUG=express-locallibrary-tutorial:* & npm start
     ps: $env:DEBUG='express-locallibrary-tutorial:*'; npm start

I have not tired the Linux commands because I am using windows right now.

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #1580
- Fixes flaws
- Uses standard note format

### CODE REVIEW

File: files/en-us/learn/server-side/express_nodejs/skeleton_website/index.html [hunk 1]
```diff
@@ -191,9 +196,16 @@ <h2 id="Running_the_skeleton_website">Running the skeleton website</h2>
...
   <ul>
-   <li>On Windows, use this command:
+   <li>For the Windows CMD prompt, use this command:
     <pre class="brush: bash notranslate">SET DEBUG=express-locallibrary-tutorial:* &amp; npm start</pre>
    </li>
+   <li>On Windows Powershell, use this command:
+    <pre class="brush: bash notranslate">$ENV:DEBUG = "express-locallibrary-tutorial:*"; npm start</pre>
+    <div class="notecard note">
+      <h4>Note</h4>
+        <p>Powershell command are not covered further in this tutorial ("Windows" commands givens are intended for use with the Windows CMD prompt.)</p>
```
Comment (chrisdavidmills) [inline files/en-us/learn/server-side/express_nodejs/skeleton_website/index.html:206]: "command" -> "commands"

and later on 

"givens" -> "given"

File: files/en-us/learn/server-side/express_nodejs/skeleton_website/index.html [hunk 2]
```diff
@@ -191,9 +196,16 @@ <h2 id="Running_the_skeleton_website">Running the skeleton website</h2>
  </li>
  <li>Then run the application.
   <ul>
-   <li>On Windows, use this command:
+   <li>For the Windows CMD prompt, use this command:
```
Comment (chrisdavidmills) [inline files/en-us/learn/server-side/express_nodejs/skeleton_website/index.html:199]: "For the..." sounds a bit odd here. "On the..." ?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Couple of nits, but mostly looks fine.

File: files/en-us/learn/server-side/express_nodejs/skeleton_website/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/learn/server-side/express_nodejs/skeleton_website/index.html]: Yoiks. Fixed.

**Comment by hamishwillee** [comment]
Thanks @chrisdavidmills , Nits-fixed :-)

**Review by chrisdavidmills — APPROVED** [review]
Cool, LGTM!

**Pull request merged**